configuration/ConfigHandler.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef CONFIG_CONFIGHANDLER_HPP
+#define CONFIG_CONFIGHANDLER_HPP
+
+#include <string>
+#include <map>
+
+namespace Configuration
+{
+
+    /**
+     * @brief Handles the server configuration
+     * by parsing the config.yml file
+     */
+    class ConfigHandler
+    {
+    public:
+        ConfigHandler(std::string);
+        ~ConfigHandler();
+
+        std::string getIP(void);
+        std::string getMotd(void);
+        std::string getPort(void);
+        std::string getMaxPlayers(void);
+
+    private:
+        std::map <std::string, std::string> _configFile;
```

Comment:
Maybe you can use an unordered_map instead of a map

Reply:
This would indeed get way better performance but it is only a config reader, so it's not that important